examples/apps/cchmc_ped_abd_ct_seg_app/segmentation_contour_operator.py

```
@@ -0,0 +1,205 @@
+                label_image = np.transpose(label_image, (1, 0))
+            else:
+                raise ValueError(
+                    f"Unsupported number of dimensions in label image: {label_image.ndim}  Expected 2 or 3."
+                )
+            # Unsqueeze to add channel dimension
+            label_image = np.expand_dims(label_image, axis=0)
+        elif isinstance(label_image, MetaTensor):
+            _meta = label_image  # retain MetaTensor reference for fallback
+            try:
+                label_image = xp.asarray(_meta)  # Direct conversion to cupy array if possible
+            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)